.circleci/config.yml

```diff
@@ -1512,6 +1512,8 @@ workflows:
           name: cannon-go-lint-and-test-<<matrix.mips_word_size>>-bit
           requires:
             - contracts-bedrock-build
+          skip_slow_tests: false
+          notify: true
```

Comment:
Could you also add `mentions: "@proofs-team"`?

Reply:
This is just to be extra explicit?  Looks like it defaults to proofs-team [here](https://github.com/ethereum-optimism/optimism/blob/971eb82eecb9d158788e463d15fbc8d42df7388e/.circleci/config.yml#L245-L249).